app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100, hidden: bool = False):
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        self.alive.append(self)
```

Comment:
Use the class name to get access to the class attributes:
https://github.com/mate-academy/py-herbivores-and-carnivores/blob/master/checklist.md